Be more forgiving with available CPU scaling governors assert

erock2112 opened the pull request:
Rather than enforcing that all of the device's scaling governors are
in the KNOWN_CPU_SCALING_GOVERNOR_VALUES list, enforce that at least
one of them is in the list. We already enforce that the desired governor
is in the known list when we try to set it, so we just need to ensure
that there's at least one available which is valid.

erock2112 commented:
This is needed for Skia's Galaxy S3 bot, which fails this assertion due to the presence of its pegasusq governor: https://chromium-swarm.appspot.com/restricted/bot/skia-rpi-006

maruel commented on adb/contrib/high.py:
```diff
@@ -146,8 +146,8 @@ def _InitCache(device):
       available_governors = sorted(i for i in out.split())
-      assert set(available_governors).issubset(
-          KNOWN_CPU_SCALING_GOVERNOR_VALUES), available_governors
+      assert len(set(available_governors).intersect(
```
Actually, len() and > 0 are not needed.

This will work:
assert set(available_governors).intersect(KNOWN_CPU_SCALING_GOVERNOR_VALUES), available_governors

erock2112 replied:
Done in subsequent commit.

erock2112 committed:
Be more forgiving with available CPU scaling governors assert

Rather than enforcing that all of the device's scaling governors are
in the KNOWN_CPU_SCALING_GOVERNOR_VALUES list, enforce that at least
one of them is in the list. We already enforce that the desired governor
is in the known list when we try to set it, so we just need to ensure
that there's at least one available which is valid.
